pages/cart.js

```diff
@@ -9,7 +9,17 @@ module.exports = {
   confirmMyOrderButton: { xpath: "//p[@class='cart_navigation clearfix']//button[@type='submit']" },
   shippingPrice: { xpath: "//span[@class='ajax_cart_shipping_cost']" },
   totalAmount: { xpath: "//span[@class='ajax_block_cart_total']" },
-  // amountPrice: { xpath: "//span[@class='price']" },
+  orderReference: { xpath: "//div[@class='box']" },
+  textShopSummary: ('SHOPPING-CART SUMMARY'),
+  textAddresses: ('ADDRESSES'),
+  textShipping: ('SHIPPING'),
+  textChoosePayMethod: ('PLEASE CHOOSE YOUR PAYMENT METHOD'),
+  textOrderSummary: ('ORDER SUMMARY'),
+  textOrderConfirm: ('ORDER CONFIRMATION'),
```

Comment:
use naming format like %itemName%%itemType%, so your text locators will be `shippingText`